schemas/pages/home-page.ts

```diff
@@ -41,8 +49,8 @@ const homePage = defineType({
       name: "twoImagesSection",
       type: "object",
       fields: [
-        // TODO: #376 Modificar el campo 'description' en el esquema `home-page`
         ...textFields,
+        ...descriptionBlockField,
```

Comment:
Hi @alexxuss1996, I just saw that Sanity didn't like having two "description" fields (you can check [http://localhost:3000/studio/](http://localhost:3000/studio/) to see the error).

So, let's just add a 'title' field into `descriptionBlockField`. And then remove `...textFields` from here.. (and line 107)

Also, to make the `block` field type work, we need to have it within an array:
```ts
  defineField({
    title: "Descripción",
    name: "description",
    type: "array",
    of: [{ type: "block" }],
  }),
```

Let me know if you have any other questions!

Reply:
Oh, ok, got it! @TonyMckes